### DISCUSSION TITLE
ABB Terra config

### DISCUSSION THREAD

**Discussion by pke3** [lbbrhzn/ocpp#988] [Q&A]
Hi all, I’ve just installed an ABB Terra wallbox and I’m struggling to configure the OCPP server. Can someone that own this wallbox help me to understand if I need the support of my installer (I have read some topic regarding some web portal of ABB where only the authorized installer have access, mine of course not) or can I do the configuration  by myself with the Terra Config app? If the latter, what I need to specify in the app, I’ve tried with ws://server:port but I receive back an error during the push of the config. Also, in the integration in HA for the charger entity I need to put the SN of the wallbox or can simply leave “charger” as value?

Thank you 😊

**Comment by wlphi** [comment]
You need access to the terra config app to set the OCPP server. you can get credentials from ABB if you confirm to be a trained professional. For HA config you can use the name set in the terra config app as an identifier.
